docs/en/get_started.md

```diff
@@ -0,0 +1,268 @@
+
+1. Create a conda virtual environment and activate it.
+
+   ```shell
+   conda create -n open-mmlab python=3.9 -y
+   conda activate open-mmlab
+   ```
+
+2. Install PyTorch and torchvision following the [official instructions](https://pytorch.org/). Here we use PyTorch 1.10.0 and CUDA 11.1.
+You may also switch to other version by specifying the version number.
+
+   **Install with conda**
+   ```shell
+   conda install pytorch=1.11.0 torchvision cudatoolkit=11.3 -c pytorch
+   ```
+
+   **Install with pip**
+   ```shell
+   pip install torch==1.11.0+cu113 torchvision==0.12.0+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
+   ```
+
+3. Install mmcv-full, we recommend you to install the pre-build package as below.
+
+   ```shell
+   pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/{cu_version}/{torch_version}/index.html
+   ```
+
+   mmcv-full is only compiled on PyTorch 1.x.0 because the compatibility usually holds between 1.x.0 and 1.x.1. If your PyTorch version is 1.x.1, you can install mmcv-full compiled with PyTorch 1.x.0 and it usually works well.
+
+
+   ```shell
+   # We can ignore the micro version of PyTorch
+   pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.11.0/index.html
+   ```
+
+   See [here](https://mmcv.readthedocs.io/en/latest/get_started/installation.html) for different versions of MMCV compatible to different PyTorch and CUDA versions.
+   Optionally you can choose to compile mmcv from source by the following command
+
+   ```shell
+   git clone https://github.com/open-mmlab/mmcv.git
+   cd mmcv
+   MMCV_WITH_OPS=1 pip install -e .  # package mmcv-full, which contains cuda ops, will be installed after this step
+   # pip install -e .  # package mmcv, which contains no cuda ops, will be installed after this step
+   cd ..
+   ```
+
+   **Important**: You need to run pip uninstall mmcv first if you have mmcv installed. Because if mmcv and mmcv-full are both installed, there will be ModuleNotFoundError.
+
+4. Install MMEngine
+
+   ```shell
+   pip install mmengine
+   ```
+
+5. Install MMDetection
+
+   ```shell
+   pip install mmdet
+   ```
+
+   Optionally, you can also build MMDetection from source in case you want to modify the code:
+
+   ```shell
+   git clone https://github.com/open-mmlab/mmdetection.git
+   cd mmdetection
+   pip install -r requirements/build.txt
+   pip install -v -e .  # or "python setup.py develop"
+   ```
+
+6. Clone the MMTracking repository.
+
+   ```shell
+   git clone https://github.com/open-mmlab/mmtracking.git
+   cd mmtracking
+   ```
+
+7. Install build requirements and then install MMTracking.
+
+   ```shell
+   pip install -r requirements/build.txt
+   pip install -v -e .  # or "python setup.py develop"
+   ```
+
+8. Install extra dependencies
+
+- For MOT evaluation (required):
+
+  ```shell
+  pip install git+https://github.com/JonathonLuiten/TrackEval.git
+  ```
+
+- For VOT evaluation (optional)
```

Comment:
Maybe we should remind the user that, we should install vot toolkit first, otherwise there may be an error.

Reply:
There are no numpy conficts  if we  perform the `pip install -r requirements/build.txt` before installing mmtrack. 
I run the install pipeline above and it's OK.